src/protocolgame.cpp

```diff
@@ -2039,7 +2039,6 @@ void ProtocolGame::sendSaleItemList(const std::list<ShopInfo>& shop)
 
 	NetworkMessage msg;
 	msg.addByte(0x7B);
-	msg.add<uint64_t>(playerBank + playerMoney); // deprecated and ignored by QT client. OTClient still uses it.
```

Comment:
why was this removed?

Reply:
I'm not sure about its use, I don't know if it's still used by otclient.

Reply:
OTClient has a system to handle multiple protocols, they can skip this field for specific versions, it's OK to change

Reply:
Mehah's OTC no longer use that line, it can be removed safely